lang/python/python-jsonschema/Makefile

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright (C) 2020 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-jsonschema
+PKG_VERSION:=3.2.0
+PKG_RELEASE:=1
+
+PYPI_NAME:=jsonschema
+PKG_HASH:=c8a85b28d377cc7737e46e2d9f2b4f44ee3c0e1deac6bf46ddefc7187d30797a
+
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
```

Comment:
This is wrong.

It's COPYING instead of LICENSE. ;)
https://github.com/Julian/jsonschema/blob/master/COPYING

Reply:
My mistake. Fixed now.

Reply:
Fixed